swift_browser_ui_frontend/src/components/ObjectTable.vue

```diff
@@ -79,15 +79,19 @@
       id="optionsbar"
       justify="space-between"
     >
-      <div class="search">
-        <b-input
-          v-model="searchQuery"
-          :placeholder="$t('message.objects.filterBy')"
-          type="search"
-          icon="filter-variant"
+      <c-text-field
+        id="search"
+        v-csc-model="searchQuery"
+        name="search"
+        custom-class="has-text-dark"
+        :placeholder="$t('message.objects.filterBy')"
```

Comment:
This `custom-class` is also not part of `c-text-field`

Reply:
removed